vortex-serde/src/layouts/reader/mod.rs

```diff
@@ -0,0 +1,12 @@
+use projections::Projection;
+
+pub mod batch;
+pub mod buffered;
+pub mod builder;
+pub mod filtering;
+pub mod projections;
+pub mod schema;
+pub mod stream;
+
+const DEFAULT_BATCH_SIZE: usize = 65536;
+const DEFAULT_PROJECTION: Projection = Projection::All;
```

Comment:
not sure this is necessery - its basically `Projection::default` which is pretty clear IMO